squarelet/templates/base.html

```diff
@@ -27,13 +28,13 @@
 
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
     {% block css %}
-      <!-- Your stuff: Third-party CSS libraries go here -->
-      {% compress css %}
-        <!-- This file stores project-specific CSS -->
-        <link href="{% static 'css/main.css' %}" rel="stylesheet">
-        <link href="{% static 'css/autocomplete.css' %}" rel="stylesheet">
-        <link href="{% static 'hijack/hijack-styles.css' %}" rel="stylesheet">
-      {% endcompress %}
+    <!-- Your stuff: Third-party CSS libraries go here -->
+    {% compress css %}
+    <!-- This file stores project-specific CSS -->
+    {# <link href="{% static 'css/main.css' %}" rel="stylesheet"> #}
+    {# <link href="{% static 'css/autocomplete.css' %}" rel="stylesheet"> #}
+    <link href="{% static 'hijack/hijack-styles.css' %}" rel="stylesheet">
+    {% endcompress %}
```

Comment:
It would be really helpful to add a code comment explaining how the `django_vite` plugin is loading CSS and JS files into the base template. I think I get the gist of it from their documentation, but a little more explanation will help others dropping into this codebase.

We should also consider removing these commented-out obsolete bits of asset loading.